tests/test_model_attributes.py

```diff
@@ -248,3 +249,73 @@ def test_attribute_inclusion_case_sensitivity():
             "urn:ietf:params:scim:schemas:core:2.0:User",
         ],
     }
+
+
+def test_attribute_inclusion_schema_extensions():
+    """Verifies that attributes from schema extensions work."""
+    user = User[EnterpriseUser].model_validate(
+        {
+            "userName": "foobar",
+            "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User": {
+                "employeeNumber": "12345"
+            },
+        }
+    )
+    assert user.model_dump(
+        scim_ctx=Context.RESOURCE_QUERY_RESPONSE,
+        attributes=[
+            "urn:ietf:params:scim:schemas:core:2.0:User:userName",
+        ],
+    ) == {
+        "schemas": [
+            "urn:ietf:params:scim:schemas:core:2.0:User",
+            "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User",
+        ],
+        "userName": "foobar",
+        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User": {
+            "schemas": ["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"],
```

Comment:
I am not sure `schemas` should be included inside the extension attribute.

For instance, [RFC7643 §8.3](https://datatracker.ietf.org/doc/html/rfc7643#section-8.3) does not include it. What do you think?

Reply:
Indeed, you are right. I don't know how to fix that though.

Also, schema extensions always have `id`, `meta` and `external_id`, which they also should not have :worried:

Reply:
Ok I will take a look at this.